extended-statistics/src/test/java/org/infinispan/stats/logic/OptimisticLockingTxClusterExtendedStatisticLogicTest.java

```diff
@@ -126,7 +124,6 @@ public void testWriteSkewOnOwner() throws Exception {
       doWriteSkewTest(true);
    }
 
-   @Test (groups = "unstable", description = "https://issues.jboss.org/browse/ISPN-3341")
```

Comment:
Sorry, I meant to enable this just so I could get more results from CI, but forgot about it.
Your patch looks good, I just changed the code to use `eventuallyEquals()`. (My new favourite!)